data-loader/cli/src/main/java/com/scalar/db/dataloader/cli/command/dataimport/ImportCommandOptions.java

```diff
@@ -69,11 +73,19 @@ public class ImportCommandOptions {
       description = "Path to the JSON control file for data mapping")
   protected String controlFilePath;
 
+  // Deprecated option - kept for backward compatibility
```

Comment:
Can we mention in the comment which release it was deprecated in?
Also, should we specify which future release we plan to delete the option from?
For reference, in another part of the code, we mention this piece of information. 
https://github.com/scalar-labs/scalardb/blob/8105889181a0fff46aeb5568aa5dc38867163e20/core/src/main/java/com/scalar/db/api/Get.java#L115-L119